docs/src/pages/components/lists/lists.md

```diff
@@ -86,6 +86,12 @@ Unfortunately it's [not implemented](https://caniuse.com/#search=sticky) by all
 
 {{"demo": "pages/components/lists/InsetList.js", "bg": true}}
 
+## Disabled Gutters
```

Comment:
```suggestion
## Disabled gutters
```

Reply:
Renamed demo to `Gutterless List` to follow the pattern used by most List demos